plugins/outputs/http/README.md

```diff
@@ -0,0 +1,36 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`
+
+It requires a `url` name.
+
+### Configuration:
+
+```toml
+# Send telegraf metrics to HTTP Server(s)
+[[outputs.graylog]]
```

Comment:
In theory, there's nothing preventing you from sending a request body in a GET request. The HTTP protocol allows it, but have no defined semantics, so it's up to you to document what exactly is going to happen when a client sends a GET payload. For instance, client have to define if parameters in a JSON body are equivalent to querystring parameters or something else entirely.
So, I thought it would be better to provide a combination of POST Method and Request Body. We thought that it would be better to explicitly set the data type of the request body to be processed in the client side by informing the data format of the serializer through Content-Type.